bowling/file.cpp

```diff
@@ -1 +1,68 @@
 #include "file.hpp"
+
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <iterator>
+#include <sstream>
+
+std::vector<std::pair<int, int>> File::getScores() const {
+    return score_;
+}
+
+std::string File::getResults() const {
+    return results_;
+}
+
+void File::splitString(const std::string& str, std::vector<std::string>& vec, char delim = '|') {

```

Comment:
would't it be better idea to set delim as const char in file.hpp ?
i'm not sure if we would be using other character than "|"